script/package-lock.json

```diff
@@ -8,6 +8,202 @@
       "resolved": "https://registry.npmjs.org/7zip-bin/-/7zip-bin-4.0.2.tgz",
       "integrity": "sha512-XtGk+IF57pr852UK1AhQJXqmm1WmSgS5uISL+LPs0z/iAxXouMvdlLJrHPeukP6gd7yR2rDTMSMkHNODgwIq7A=="
```

Comment:
https://status.circleci.com/

Reply:
What is this spam? I reported this to GitHub.